### ISSUE TITLE
Inquiry about Testing for Chinese Dialects

### ISSUE THREAD

**Issue by YANG627** [DigitalPhonetics/IMS-Toucan#177]
Hi,

I'm really impressed with your work on the TTS system for 7000+ languages!

I have a question about testing for Chinese dialects like Cantonese (粤语) and Xiang (湘语):

How did you evaluate and perform inference for these dialects?
Could you share some sample texts used for testing these dialects for reference?
Thanks a lot!

**Comment by Flux9665** [comment]
We did not perform testing on all languages we support, we only sampled a few languages to make sure that the system works generally. Cantonese and Xiang were not among the languages we tested. We also have no data for either of the two, so they are among the very challenging languages we only approximate with a predicted language embedding. The system would likely perform much better if it was finetuned on some data from the respective language.

Generally, the quality of the TTS varies greatly across languages. We hope to improve on this initial system. I also noticed that in our text processing, we were not handling the 6 tones of Cantonese properly, this is fixed in the next version, that is set to be released in one or two weeks. For Xiang we unfortunately don't even have a phonemizer, so we are using Transphone, which does not output tone symbols. I expect Xiang to sound very incorrect because of that.
